packages/klevu-ui/src/components/klevu-product/klevu-product.css

```diff
@@ -40,15 +40,15 @@
   display: flex;
   flex-direction: column;
   max-width: var(--width);
-  gap: var(--klevu-spacing-06);
+  gap: var(--klevu-product-details-spacing,--klevu-spacing-06);
```

Comment:
the standard way used across components is to declare a local variable with above value and assign local variable to the style, pls follow that. same for next one.